[BUG] Marking messages as read doesn't work on PostgreSQL.

**Describe the bug**
Marking messages as read doesn't work on PostgreSQL.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Channels'
2. Click on 'Mark all as Read'
3. Red dot on Channels icon doesn't disappear, no new entries on read_messages table

**Expected behavior**
Messages to be marked as read

**Environment:**
* Host OS: Linux
* Deployment type: Docker, tag latest (3.11.1)
* Database backend: PostgreSQL
* Node type: Seeed Xiao NRF52840 Kit
* Node Connection Type: ble-bridge


**Additional context**
```
[meshmonitor] [ERROR] Failed to mark channel 0 messages as read: DrizzleQueryError: Failed query: 
[meshmonitor] INSERT INTO read_messages ("messageId", "userId", "readAt")
[meshmonitor] SELECT id, $1, $2 FROM messages
[meshmonitor] WHERE channel = $3
[meshmonitor] AND portnum = 1
[meshmonitor] ON CONFLICT ("messageId", "userId") DO NOTHING
[meshmonitor] 
[meshmonitor] params: 2,1774996695143,0
[meshmonitor] at NodePgPreparedQuery.queryWithCache (file:///app/node_modules/drizzle-orm/pg-core/session.js:41:15)
[meshmonitor] at process.processTicksAndRejections (node:internal/process/task_queues:104:5)
[meshmonitor] at async NotificationsRepository.markChannelMessagesAsRead (file:///app/dist/db/repositories/notifications.js:314:30) {
[meshmonitor] query: '\n' +
[meshmonitor] '            INSERT INTO read_messages ("messageId", "userId", "readAt")\n' +
[meshmonitor] '            SELECT id, $1, $2 FROM messages\n' +
[meshmonitor] '            WHERE channel = $3\n' +
[meshmonitor] '              AND portnum = 1\n' +
[meshmonitor] '            ON CONFLICT ("messageId", "userId") DO NOTHING\n' +
[meshmonitor] '          ',
[meshmonitor] params: [ 2, 1774996695143, 0 ],
[meshmonitor] cause: error: there is no unique or exclusion constraint matching the ON CONFLICT specification
```

--- a/src/db/repositories/notifications.ts
+++ b/src/db/repositories/notifications.ts
@@ -503,23 +503,29 @@ export class NotificationsRepository extends BaseRepository {
         if (beforeTimestamp !== undefined) {
           result = await db.execute(sql`
             INSERT INTO read_messages (${this.col('messageId')}, ${this.col('userId')}, ${this.col('readAt')})
-            SELECT id, ${effectiveUserId}, ${now} FROM messages
-            WHERE ((${this.col('fromNodeId')} = ${localNodeId} AND ${this.col('toNodeId')} = ${remoteNodeId})
-                OR (${this.col('fromNodeId')} = ${remoteNodeId} AND ${this.col('toNodeId')} = ${localNodeId}))
-              AND portnum = 1
-              AND channel = -1
-              AND timestamp <= ${beforeTimestamp}
-            ON CONFLICT (${this.col('messageId')}, ${this.col('userId')}) DO NOTHING
+            SELECT m.id, ${effectiveUserId}, ${now} FROM messages m
+            WHERE ((m.${this.col('fromNodeId')} = ${localNodeId} AND m.${this.col('toNodeId')} = ${remoteNodeId})
+                OR (m.${this.col('fromNodeId')} = ${remoteNodeId} AND m.${this.col('toNodeId')} = ${localNodeId}))
+              AND m.portnum = 1
+              AND m.channel = -1
+              AND m.timestamp <= ${beforeTimestamp}
+              AND NOT EXISTS (
+                SELECT 1 FROM read_messages rm
+                WHERE rm.${this.col('messageId')} = m.id AND rm.${this.col('userId')} = ${effectiveUserId}
+              )
           `);
         } else {
           result = await db.execute(sql`
             INSERT INTO read_messages (${this.col('messageId')}, ${this.col('userId')}, ${this.col('readAt')})
-            SELECT id, ${effectiveUserId}, ${now} FROM messages
-            WHERE ((${this.col('fromNodeId')} = ${localNodeId} AND ${this.col('toNodeId')} = ${remoteNodeId})
-                OR (${this.col('fromNodeId')} = ${remoteNodeId} AND ${this.col('toNodeId')} = ${localNodeId}))
-              AND portnum = 1
-              AND channel = -1
-            ON CONFLICT (${this.col('messageId')}, ${this.col('userId')}) DO NOTHING
+            SELECT m.id, ${effectiveUserId}, ${now} FROM messages m
+            WHERE ((m.${this.col('fromNodeId')} = ${localNodeId} AND m.${this.col('toNodeId')} = ${remoteNodeId})
+                OR (m.${this.col('fromNodeId')} = ${remoteNodeId} AND m.${this.col('toNodeId')} = ${localNodeId}))
+              AND m.portnum = 1
+              AND m.channel = -1
+              AND NOT EXISTS (
+                SELECT 1 FROM read_messages rm
+                WHERE rm.${this.col('messageId')} = m.id AND rm.${this.col('userId')} = ${effectiveUserId}
+              )
           `);
         }
         return Number(result.rowCount ?? 0);
@@ -605,11 +611,14 @@ export class NotificationsRepository extends BaseRepository {
         const db = this.getPostgresDb();
         const result = await db.execute(sql`
           INSERT INTO read_messages (${this.col('messageId')}, ${this.col('userId')}, ${this.col('readAt')})
-          SELECT id, ${effectiveUserId}, ${now} FROM messages
-          WHERE (${this.col('fromNodeId')} = ${localNodeId} OR ${this.col('toNodeId')} = ${localNodeId})
-            AND portnum = 1
-            AND channel = -1
-          ON CONFLICT (${this.col('messageId')}, ${this.col('userId')}) DO NOTHING
+          SELECT m.id, ${effectiveUserId}, ${now} FROM messages m
+          WHERE (m.${this.col('fromNodeId')} = ${localNodeId} OR m.${this.col('toNodeId')} = ${localNodeId})
+            AND m.portnum = 1
+            AND m.channel = -1
+            AND NOT EXISTS (
+              SELECT 1 FROM read_messages rm
+              WHERE rm.${this.col('messageId')} = m.id AND rm.${this.col('userId')} = ${effectiveUserId}
+            )
         `);
         return Number(result.rowCount ?? 0);
       } else {
@@ -662,8 +671,11 @@ export class NotificationsRepository extends BaseRepository {
         for (const messageId of messageIds) {
           await db.execute(sql`
             INSERT INTO read_messages (${this.col('messageId')}, ${this.col('userId')}, ${this.col('readAt')})
-            VALUES (${messageId}, ${effectiveUserId}, ${now})
-            ON CONFLICT (${this.col('messageId')}, ${this.col('userId')}) DO NOTHING
+            SELECT ${messageId}, ${effectiveUserId}, ${now}
+            WHERE NOT EXISTS (
+              SELECT 1 FROM read_messages rm
+              WHERE rm.${this.col('messageId')} = ${messageId} AND rm.${this.col('userId')} = ${effectiveUserId}
+            )
           `);
         }
       } else {

